src/huggingface_hub/commands/download.py

```diff
@@ -94,21 +93,17 @@ def register_subcommand(parser: _SubParsersAction):
         download_parser.add_argument(
             "--local-dir-use-symlinks",
             choices=["auto", "True", "False"],
-            default="auto",
-            help=(
-                "To be used with `local_dir`. If set to 'auto', the cache directory will be used and the file will be"
-                " either duplicated or symlinked to the local directory depending on its size. It set to `True`, a"
-                " symlink will be created, no matter the file size. If set to `False`, the file will either be"
-                " duplicated from cache (if already exists) or downloaded from the Hub and not cached."
-            ),
+            help=("Deprecated and ignored. Downloading to a local directory does not use symlinks anymore."),
         )
         download_parser.add_argument(
             "--force-download",
             action="store_true",
             help="If True, the files will be downloaded even if they are already cached.",
         )
         download_parser.add_argument(
-            "--resume-download", action="store_true", help="If True, resume a previously interrupted download."
+            "--resume-download",
+            action="store_true",
+            help="Deprecated and ignored. Downloading a file to local dir always attempt to resume previously interrupted download.",
```

Comment:
```suggestion
            help="Deprecated and ignored. Downloading a file to local dir always attempts to resume previously interrupted downloads (unless hf-transfer is enabled).",
```

I wonder if we should be more clear about hf-transfer, I was using `HF_HUB_ENABLE_HF_TRANSFER=1` and only realized this behaviour after looking at the pr.

Reply:
That's also why I stopped to promote `hf_transfer` when running the download CLI. `hf_transfer` is more limited in term of features and in particular is not able to resume an incomplete file (by design). I might update the `hf-transfer` docs at some point to explain more the limitations but in the meantime, let's not promote it at all. (+I added your changes)